parse_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/catatsuy/movabletype"
+	. "github.com/otoyo/movabletype"
```

Comment:
Please don't change package name.

Reply:
Sorry, I fixed it at ae81c9a . Thank you.